src/to-words.ts

```diff
@@ -46,6 +46,8 @@ export class ToWords {
         return require('./locales/en-MU').Locale;
       case 'en-US':
         return require('./locales/en-US').Locale;
+      case 'en-NG':
+        return require('./locales/en-NG').Locale;  
```

Comment:
@dueka Please help fix the lint issues. There seems to be some extra space at the end of this line.

Reply:
I have fixed the linting issue and pushed it.